### PR TITLE
Generalised specific type of button

### LINKED ISSUE #27: Generalise specific type of button
We literally use a specific set of classes for button and it is becoming repetitive.
We use
```
btn bg-zinc-900 hover:bg-zinc-800 text-white
```
Bake that into existing utility-patterns css file and cleanup from following HTML while ensuring nothing breaks. (global search `bg-zinc-800` to find latest results. these might be obsolete.
```
11 results - 8 files

src/components/abstract/ConfirmationBox.vue:
  55          </button>
  56:         <button class="btn-sm bg-zinc-900 hover:bg-zinc-800 text-white" @click.stop="confirm(close)">
  57            {{ confirmLabel }}

src/components/abstract/OptionsSelectorDropdown.vue:
  51            <li>
  52:             <button class="btn-xs bg-zinc-900 hover:bg-zinc-800 text-white" @click="makeConfirmAction(close)">
  53                {{ confirmLabel }}

src/components/admin/integration/hooks/Form.vue:
  97  
  98:       <button class="flex btn btn-sm mt-4 bg-zinc-900 hover:bg-zinc-800 text-white" :disabled="isSubmitting" type="submit">
  99          {{ isUpdating ? "Save" : "Create" }}

src/components/admin/postings/EmptyState.vue:
  50      </div>
  51:     <NuxtLink to="/admin/postings/edit" class="btn bg-zinc-900 hover:bg-zinc-800 text-white">
  52        <Icon name="ic:baseline-plus" class="w-5 h-5"/>

src/components/admin/settings/general/Update.vue:
  117            <div class="flex w-full justify-start mb-10 mt-4">
  118:             <button class="btn bg-zinc-900 hover:bg-zinc-800 text-white" @click="onSubmit" :disabled="isSubmitting">
  119                {{ saveLabel }}

  163            <div class="flex w-full justify-start mb-10 mt-4">
  164:             <button class="btn bg-zinc-900 hover:bg-zinc-800 text-white" @click="onSubmit" :disabled="isSubmitting">
  165                {{ saveLabel }}

src/components/admin/settings/members/AddAction.vue:
  39      <template #input="{ open }">
  40:       <button class="btn bg-zinc-800 hover:bg-zinc-800 text-white text-xs" @click="open">
  41          <span class="mr-2">Add Member</span>

src/pages/admin/postings/edit.vue:
  129          </div>
  130:         <button class="btn bg-zinc-900 hover:bg-zinc-800 text-white flex space-x-2" @click="onSubmit">
  131            <Icon name="lets-icons:save" class="w-4 h-4" />

src/pages/postings/[id].vue:
  120                </div>
  121:               <button class="btn w-full bg-zinc-900 hover:bg-zinc-800 text-white" @click="apply" :disabled="isApplying"
  122                  v-else>Apply Today

  162  
  163:               <button class="btn w-full bg-zinc-900 hover:bg-zinc-800 text-white" @click="apply" :disabled="isApplying"
  164                  v-else>Apply Today

```

### PR DESCRIPTION
**Pull request by Pushkarm029**
Resolves #27

### CODE REVIEW

File: src/assets/css/additional-styles/utility-patterns.css [hunk 1]
```diff
@@ -49,6 +49,10 @@
   @apply px-2 py-0.5;
 }
 
+.btn-primary {
```
Comment (Pushkarm029) [inline src/assets/css/additional-styles/utility-patterns.css:52]: I think we need a better name??

Comment (harshil15999) [reply src/assets/css/additional-styles/utility-patterns.css:52]: Ok what would you suggest?

Comment (Pushkarm029) [reply src/assets/css/additional-styles/utility-patterns.css:52]: how `.bg-zinc` sounds?

Comment (amandesai01) [reply src/assets/css/additional-styles/utility-patterns.css:52]: `btn-primary` is correct.

**Pull request merged**